### DISCUSSION TITLE
How to create scaffold app using quasar cli with the latest beta quasar?

### DISCUSSION THREAD

**Discussion by dmitry** [quasarframework/quasar#9531] [General - Components / Directives / etc] [answered]
I'd like to install beta version of quasar with vue 3, latest components from the vue3-work branch, use the quasar/app v3-beta. How can I do that?

- Tried to install `@quasar/cli` but it's on version 1.2 which is creating quasar v1 and quasar/app v2. 
- Tried to upgrade `@quasar` and `@quasar/app` to the `@next` versions, and then run `quasar create app`, but it is still creating with an older versions of quasar base/app.
- Tried to google to find out how to install / create scaffold app with the beta versions, but without luck. 

Seems like beta version of `@quasar/cli` isn't published yet on the npm (https://github.com/quasarframework/quasar/tree/vue3-work/cli), but how do I can install it then or there is the only way at the moment to run an upgrade from the previous version?

PS. After some tryouts, I found `quasar create app --branch next`. Is this command still valid with `@quasar/cli` v1.2? `quasar info` shows it's using the latest npm packages (beta versions), but does it generates correct filestructure which fits to the quasar beta?

**Comment by metalsadman** [comment]
remove your old @quasar/cli ie. `npm uninstall -g @quasar/cli` then reinstall https://next.quasar.dev/quasar-cli/installation#introduction.

**Reply by dmitry** [reply]
@metalsadman that's what I've tried, and it installed quasar/cli which allows me only to create app with stable version, not beta. In guide you just linked `npm install -g @quasar/cli` installs v1.2. Seems like the only way at the moment to do that, is to force `quasar-create` to use `--branch next`.

**Comment by rstoenescu** [comment, marked as answer]
`@quasar/cli` is the same for Quasar v0.x, v1 and Quasar v2 and any future versions. No need to touch it. Just keep it up to date for security purposes and its new features. Whenever you scaffold a new project with it, it downloads the template from the Internet (based on the branch param) so supporting any version of Quasar does not require updating `@quasar/cli`.

What I don't understand is why having to google anything, since the information is in docs. https://next.quasar.dev/quasar-cli/installation --- or the Getting started menu section with Quasar CLI https://next.quasar.dev/start/quasar-cli
